### PR TITLE
CI

### PR DESCRIPTION
**Pull request by pzl**
## Change Summary

Begin CI instructions

### CODE REVIEW

**Comment by pzl** [comment]
@elasticmachine do your job, man

File: package/endpoint/data_stream/network/_dev/test/pipeline/test-common-config.yml [hunk 1]
```diff
@@ -1,2 +1,5 @@
 dynamic_fields:
   event.ingested: ".*"
+skip:
+  reason: geoIP broken
+  link: http://how-dare-you
```
Comment (parkiino) [inline package/endpoint/data_stream/network/_dev/test/pipeline/test-common-config.yml:5]: how dare indeed

**Review by parkiino — APPROVED** [review]
hopefully if i keep reading these, i'll get a better sense of what is happening but, 🚢  🐑  for now

**Review by kevinlog — APPROVED** [review]

**Pull request merged**